submissions/fabunmiTolulope/HTML/Productpage.html

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<meta charset="utf-8">
+	<meta name="keywords" content="banana, plantain, breeds of banana">
+	<meta name="author" content="Fabunmi Tolulope Eniola">
+	<title>Product page for Greenprint Integrated Services</title>
+</head>
+<body>
+<header>
+		<h1>Welcome to Greenprint Integrated Services</h1>
+</header>
+	<nav>
+		<div>
+			<ul>
+				<li><a href="home.html"> Home</a></li>
+				<li><a href="blog.html">Blog</a> </li>
+				<li><a href="Productpage.html">Productpage</a> </li>
+				<li><a href="aboutUs.html">Contact Us</a> </li>
+			</ul>
+		</div>
+	</nav>
+	<hr>
+<section>
+	<H3> Organic Giant Cavendish Banana</H3>
+	<img src="cavendish banana.jpg"; height="200px" width="200px";>
+	<img src="cavendish.jpg"; height="200px" width="200px";>
+	<p align="left"><strong> &#x20A6;10,000</strong></p>
```

Comment:
Also remove the align attribute of the p tag. CSS would take care of that later. For a more seamless review process, please leave all styling for CSS.